src/daemon/rpmostreed-os.c

```diff
@@ -350,42 +350,43 @@ os_handle_download_update_rpm_diff (RPMOSTreeOS *interface,
   return TRUE;
 }
 
+static inline void*
+vardict_lookup_ptr (GVariantDict  *dict,
+                    const char    *key,
+                    const char    *fmt)
+{
+  void *val = NULL;
+  g_variant_dict_lookup (dict, key, fmt, &val);
+  return val;
+}
+
+static gboolean
+vardict_lookup_bool (GVariantDict *dict,
+                     const char   *key,
+                     gboolean      dfault)
```

Comment:
Heh, I know this triggered me too. It took me a second to realize that of course, `default` is a reserved 🏴 keyword.